src/css/viewer.css

```diff
@@ -428,7 +495,7 @@ html[data-theme=night-theme] #right-page-btn:hover
 
 @media only screen and (min-width: 768px) {
   #app-container.menu--show #reading-area {
-    width: calc(100% - 400px);
```

Comment:
I think this may be required for viewport resizing.

Reply:
Kaleeg wants the sidebar to be an overlay on top of the reading area instead of having the reading area size adjusted when the sidebar is open.